### ISSUE TITLE
Upgrade Laravel Dusk Version

### ISSUE THREAD

**Issue by agmadt** [NoelDeMartin/laravel-dusk-mocking#10]
I got this problem when installing the package, can you upgrade the dusk version?
```
Problem 1
    - Can only install one of: laravel/dusk[4.0.x-dev, v5.0.1].
    - Can only install one of: laravel/dusk[v5.0.1, 4.0.x-dev].
    - Can only install one of: laravel/dusk[4.0.x-dev, v5.0.1].
    - noeldemartin/laravel-dusk-mocking v4.0.0 requires laravel/dusk ~4.0 -> satisfiable by laravel/dusk[4.0.x-dev].
    - Installation request for noeldemartin/laravel-dusk-mocking ^4.0 -> satisfiable by noeldemartin/laravel-dusk-mocking[v4.0.0].
    - Installation request for laravel/dusk (locked at v5.0.1, required as ^5.0) -> satisfiable by laravel/dusk[v5.0.1].
```

**Comment by NoelDeMartin** [comment]
I wanted to upgrade it after solving #9, but it wasn't straight forward to fix. I'll check it out and if I can't find a solution in the short term I'll make a release anyways.

**Comment by NoelDeMartin** [comment]
@agmadt I just released v5.0.0 :)

**Comment by agmadt** [comment]
@NoelDeMartin thanks for the hard work!
